.github/workflows/check-catalyst.yaml

```diff
@@ -586,7 +584,7 @@ jobs:
           C_COMPILER=$(which ${{ needs.constants.outputs[format('c_compiler.{0}', matrix.compiler)] }}) \
           CXX_COMPILER=$(which ${{ needs.constants.outputs[format('cxx_compiler.{0}', matrix.compiler)] }}) \
           COMPILER_LAUNCHER="" \
-          ENABLE_OPENQASM=ON \
+          ENABLE_LIGHTNING_KOKKOS=OFF \
```

Comment:
Is lightning.qubit needed?

Reply:
we need to build with Lightning for testing! as of the dependency of the runtime C++ test suite to Lightning. There is a separate (simple) ticket in backlog to fix this issue by splitting the C++ tests so that tests for each backend can be built & run independently from building the runtime.

Reply:
Hmm strange, yeah would be good to fix in the future 👍